build.sbt

```diff
@@ -68,7 +68,8 @@ libraryDependencies ++= Seq(
   "io.confluent" % "kafka-avro-serializer" % "5.1.2" % "provided",
   "com.uber.hoodie" % "hoodie-spark" % "0.4.7" % "provided" excludeAll(excludeHoodieTimeLineService),
   "com.uber.hoodie" % "hoodie-common" % "0.4.7" % "provided" excludeAll(excludeHoodieTimeLineService),
-  "org.apache.hive" % "hive-jdbc" % "1.2.2" % "provided" excludeAll(excludeNetty, excludeNettyAll)
+  "org.apache.hive" % "hive-jdbc" % "1.2.2" % "provided" excludeAll(excludeNetty, excludeNettyAll),
+  "org.mongodb.spark" %% "mongo-spark-connector" % "2.4.1"
```

Comment:
We will use mongo dependencies as a provided jar or package in the spark-submit command,
(for example: spark-submit --packages org.elasticsearch:elasticsearch-hadoop:6.6.1 ), and not as part of our JAR or Standalone JAR. So this dependency shouldn't appear in the build.sbt